From 134ae6223a1e1d726dcc970404734e3e52efff69 Mon Sep 17 00:00:00 2001
From: Thiago Macieira <thiago.macieira@intel.com>
Date: Tue, 3 Jul 2018 16:38:39 -0700
Subject: [PATCH 6/7] Disable the colourful PROMPT_COMMAND for chroot

The complex syntax fails to pass properly to systemd-nspawn with -E
arguments.

Signed-off-by: Thiago Macieira <thiago.macieira@intel.com>
---
 py/mockbuild/util.py | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/py/mockbuild/util.py b/py/mockbuild/util.py
index 3ba2af9..e704416 100644
--- a/py/mockbuild/util.py
+++ b/py/mockbuild/util.py
@@ -589,8 +589,8 @@ def doshell(chrootPath=None, environ=None, uid=None, gid=None, user=None, cmd=No
     log.debug("doshell: chrootPath:%s, uid:%d, gid:%d" % (chrootPath, uid, gid))
     if environ is None:
         environ = clean_env()
-    if not 'PROMPT_COMMAND' in environ:
-        environ['PROMPT_COMMAND'] = 'printf "\033]0;<mock-chroot>\007<mock-chroot>"'
+#    if not 'PROMPT_COMMAND' in environ:
+#        environ['PROMPT_COMMAND'] = 'printf "\033]0;<mock-chroot>\007<mock-chroot>"'
     if not 'SHELL' in environ:
         environ['SHELL'] = '/bin/sh'
     log.debug("doshell environment: %s", environ)
@@ -763,7 +763,7 @@ def setup_default_config_opts(unprivUid, version, pkgpythondir, unprivGid=None):
         'HOME': '/builddir',
         'HOSTNAME': 'mock',
         'PATH': '/usr/bin:/bin:/usr/sbin:/sbin',
-        'PROMPT_COMMAND': 'printf "\033]0;<mock-chroot>\007<mock-chroot>"',
+#        'PROMPT_COMMAND': 'printf "\033]0;<mock-chroot>\007<mock-chroot>"',
         'LANG': os.environ.setdefault('LANG', 'en_US.UTF-8'),
         }
 
-- 
2.18.0

